monitoring/check_collectd_mlab_test.py

```diff
@@ -0,0 +1,301 @@
+#!/usr/bin/python
+# Copyright 2014 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Unit tests for check_collectd_mlab check."""
+
+import os
+import socket
+import time
+import unittest2 as unittest
```

Comment:
This sounds familiar, but it's been a while, and revisiting choices can be helpful in case circumstances have changed.

My thinking is that having the production environment be the same as the test-environment is advantageous. And, the current M-Lab node OS (CentOS 6) uses python 2.6.6.